src/main/java/internetshop/controller/ShoppingCartController.java

```diff
@@ -11,15 +11,16 @@
 import javax.servlet.http.HttpServletResponse;
 
 public class ShoppingCartController extends HttpServlet {
```

Comment:
```suggestion
public class GetShoppingCartController extends HttpServlet {
```
давай зробимо щоб у нас неймінг контролерів мав один тип